schema_salad/tests/test_line_numbers.py

```diff
@@ -0,0 +1,146 @@
+# from parser import load_document_by_uri, save
+from pathlib import Path
+from typing import Any, MutableSequence, Optional, cast
+from urllib.parse import unquote_plus, urlparse
+
+from ruamel.yaml.comments import CommentedMap
+
+import schema_salad.tests.cwl_v1_2 as cwl_v1_2
```

Comment:
How are we keeping `schema_salad/tests/cwl_v1_2.py` up to date? Maybe it would be better to generate that when the tests are run..